lib/http-bridge/test/integration/Test/Integration/Scenario/CLI/Transactions.hs

```diff
@@ -9,49 +11,427 @@ module Test.Integration.Scenario.CLI.Transactions
+            addrs:_ <- listAddresses ctx wDest
+            let addr =
+                    encodeAddress (Proxy @HttpBridge) (getApiT $ fst $ addrs ^. #id)
+            let args = T.unpack <$>
+                    [ wSrc ^. walletId
+                    , "--payment", amt <> "@" <> addr
+                    ]
+
+            (c, out, err) <- postTransactionViaCLI "cardano-wallet" args
+            (T.unpack err) `shouldContain` errMsg
+            out `shouldBe` ""
+            if (title == "0") then
+                c `shouldBe` ExitSuccess
+            else
+                c `shouldBe` (ExitFailure 1)
+
+    describe "TRANS_CREATE_07 - False wallet ids" $ do
+        forM_ falseWalletIds $ \(title, walId) -> it title $ \ctx -> do
+            wDest <- emptyWallet ctx
+            addrs:_ <- listAddresses ctx wDest
+            let addr =
+                    encodeAddress (Proxy @HttpBridge) (getApiT $ fst $ addrs ^. #id)
+            let args = [ walId, "--payment", "12@" ++  (T.unpack addr) ]
+
+            (c, out, err) <- postTransactionViaCLI "Secure Passphrase" args
+            out `shouldBe` ""
+            if (title == "40 chars hex") then
+                (T.unpack err) `shouldContain` "I couldn't find a wallet with \
+                    \the given id: 1111111111111111111111111111111111111111"
+            else
+                (T.unpack err) `shouldContain` "wallet id should be an \
+                    \hex-encoded string of 40 characters"
+            if (title == "40 chars hex") then
+                c `shouldBe` ExitSuccess
+            else
+                c `shouldBe` (ExitFailure 1)
+
+    it "TRANSCLI_CREATE_07 - 'almost' valid walletId" $ \ctx -> do
+        wSrc <- fixtureWallet ctx
+        wDest <- emptyWallet ctx
+        addrs:_ <- listAddresses ctx wDest
+        let addr =
+                encodeAddress (Proxy @HttpBridge) (getApiT $ fst $ addrs ^. #id)
+        let args = T.unpack <$>
+                [ (T.append (wSrc ^. walletId) "0"), "--payment", "11@" <> addr ]
+
+        (c, out, err) <- postTransactionViaCLI "cardano-wallet" args
+        (T.unpack err) `shouldContain` "wallet id should be an hex-encoded\
+            \ string of 40 characters"
+        out `shouldBe` ""
+        c `shouldBe` (ExitFailure 1)
+
+    it "TRANS_CREATE_07 - Deleted wallet" $ \ctx -> do
+        wSrc <- fixtureWallet ctx
+        Exit ex <- deleteWalletViaCLI (T.unpack ( wSrc ^. walletId ))
+        ex `shouldBe` ExitSuccess
+
+        wDest <- emptyWallet ctx
+        addrs:_ <- listAddresses ctx wDest
+        let addr =
+                encodeAddress (Proxy @HttpBridge) (getApiT $ fst $ addrs ^. #id)
+        let args = T.unpack <$> [ wSrc ^. walletId, "--payment", "11@" <> addr ]
+
+        (c, out, err) <- postTransactionViaCLI "cardano-wallet" args
+        (T.unpack err) `shouldContain` "I couldn't find a wallet with \
+            \the given id: " ++ T.unpack ( wSrc ^. walletId )
+        out `shouldBe` ""
```

Comment:
I admit I haven't yet thoroughly looked at the scenarios. I suppose they replicate the ones in the API section.

Reply:
Indeed they do.